### PR TITLE
Added Maintainers to readme

### PR DESCRIPTION
**Pull request by deltadstroyer**

### CODE REVIEW

**Review by karlis-ozo — APPROVED** [review]
Nice

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 FUBot is a python Discord Bot built exclusively for the FU Discord.
 
-# Commands
+# Commands
```
Comment (Normandabald) [inline README.md:5]:
```suggestion
# Commands
```

File: README.md [hunk 2]
```diff
@@ -74,4 +74,9 @@ docker-compose down
...
 ## Contributing
 
-Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
\ No newline at end of file
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+# Maintainers
+- Guthammer
+- Karlish
+- Deltadstroyer
```
Comment (Normandabald) [inline README.md:82]: Looks good!

**Review by Normandabald — CHANGES_REQUESTED** [review]

**Review by Normandabald — APPROVED** [review]
LGTM! 🎉

**Pull request merged**